Sources/PackagePlugin/PluginInput.swift

```diff
@@ -121,6 +121,7 @@ fileprivate struct PluginInputDeserializer {
             target = SwiftSourceModuleTarget(
                 id: String(id),
                 name: wireTarget.name,
+                kind: (kind == .executable ? .executable : (kind == .test ? .test : .regular)),
```

Comment:
nit: could be done with an extension which would make this a bit more nicer to read / maintain

Reply:
Agreed.  Updating with the modified changes in the PR on main.